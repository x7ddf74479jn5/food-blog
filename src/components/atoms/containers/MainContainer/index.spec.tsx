import renderer from "react-test-renderer";

import { MainContainer } from ".";

describe("components/atoms/containers/MainContainer", () => {
  it("snapshot", () => {
    const tree = renderer.create(<MainContainer>children</MainContainer>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
