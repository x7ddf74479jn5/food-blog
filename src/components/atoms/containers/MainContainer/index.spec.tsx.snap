// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/containers/MainContainer snapshot 1`] = `
<main
  className="overflow-hidden flex-1 order-1 md:order-2 pb-12 mt-8 md:mt-16 mb-0 w-full max-w-prose"
>
  children
</main>
`;
